packages/plainjs/dist/deno/tokenizer.ts

```diff
@@ -336,6 +337,9 @@ export default class Tokenizer {
                 value: string,
                 offset: this.offset,
               });
+              this.offset += this.escapeLength;
+              this.escapeLength = 0;
```

Comment:
Please, move the reseting of the `escapedCharsByteLength` to line 304 right after reseting the string builder.